src/languages/en.js

```diff
@@ -588,12 +588,14 @@ export default {
         },
         invite: {
             invitePeople: 'Invite People',
-            invitePeoplePrompt: 'Invite a colleague to your workspace.',
+            invitePeoplePrompt: 'Invite colleagues to your workspace.',
             personalMessagePrompt: 'Add a Personal Message (Optional)',
-            enterEmailOrPhone: 'Email or Phone',
+            enterEmailOrPhone: 'Emails or Phones',
+            EmailOrPhonePlaceholder: 'Enter comma-separated emails or phones',
```

Comment:
```suggestion
            EmailOrPhonePlaceholder: 'Enter comma-separated list of emails or phone numbers',
```

Reply:
This placeholder is not fitting into a line.

Reply:
Does `Enter comma-separated emails or phone numbers` fit?

Reply:
Nope, Should I make this multiline? As it is supposed to be used to invite multiple users.

Reply:
Hmmm, okay yeah let's use the full copy `Enter comma-separated list of emails or phone numbers` and make it multi-line.